catalog-rest-service/src/main/resources/json/schema/entity/teams/user.json

```diff
@@ -95,6 +95,10 @@
       "description": "Teams that the user belongs to.",
       "$ref": "../../type/entityReference.json#/definitions/entityReferenceList"
     },
+    "groups": {
```

Comment:
@vivekratnavel let's not an additional field like this. Let's add an optional query param to list API and GET API where teamType can be passed and those will be returned.